pkg/operator/bootstrapteardown/teardown.go

```diff
@@ -2,14 +2,18 @@ package bootstrapteardown
 
 import (
 	"context"
+	"k8s.io/apimachinery/pkg/util/wait"
```

Comment:
again something is making odd choices on where these go.

Reply:
fixed